test/decorator_test.dart

```diff
@@ -9,11 +9,11 @@ import 'package:platform_detect/src/support.dart';
 
 void main() {
   group('root node CSS class injection', () {
-    Element fakeRootNode;
-    List calls;
+    Element? fakeRootNode;
```

Comment:
This looks like it could be another good candidate to be a late variable.

Reply:
Indeed it was. Updated.